lib/src/main/java/growthbook/sdk/java/multiusermode/configurations/Options.java

```diff
@@ -0,0 +1,141 @@
+package growthbook.sdk.java.multiusermode.configurations;
+
+import growthbook.sdk.java.*;
+import growthbook.sdk.java.multiusermode.usage.FeatureUsageCallbackWithUser;
+import growthbook.sdk.java.multiusermode.usage.TrackingCallbackWithUser;
+import growthbook.sdk.java.stickyBucketing.InMemoryStickyBucketServiceImpl;
+import growthbook.sdk.java.stickyBucketing.StickyBucketService;
+import lombok.Builder;
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.annotation.Nullable;
+import java.util.HashMap;
+import java.util.List;
+
+@Data
+@Slf4j
+public class Options {
+
+    @Builder
+    public Options(@Nullable Boolean enabled,
+                   Boolean isQaMode,
+                   @Nullable Boolean isCacheDisabled,
+                   Boolean allowUrlOverrides,
+                   @Nullable String url,
+                   @Nullable String apiHost,
+                   @Nullable String clientKey,
+                   @Nullable String decryptionKey,
+                   @Nullable List<String> stickyBucketIdentifierAttributes,
+                   @Nullable StickyBucketService stickyBucketService,
+                   @Nullable TrackingCallbackWithUser trackingCallBackWithUser,
+                   @Nullable FeatureUsageCallbackWithUser featureUsageCallbackWithUser,
+                   @Nullable FeatureRefreshStrategy refreshStrategy,
+                   @Nullable FeatureRefreshCallback featureRefreshCallback) {
+        this.enabled = enabled == null || enabled;
+        this.isQaMode = isQaMode != null && isQaMode;
+        this.isCacheDisabled = isCacheDisabled == null || isCacheDisabled;
```

Comment:
This looks wrong to me.  it's defaulting cache to be disabled when `isCacheDisabled` is null.  I would think we would want it enabled by default instead.